### PR TITLE
Add footer content

### PR DESCRIPTION
**Pull request by maxcell**
Eh. ¯\\_(ツ)_/¯
image

### CODE REVIEW

**Comment by maxcell** [comment]
image

Help me get good
image

**Comment by maxcell** [comment]
image

File: app/assets/stylesheets/footer/index.scss [hunk 1]
```diff
@@ -18,7 +18,8 @@ footer[role="main"] {
 
     &.icon {
         width: auto;
-        margin-right: 15px;
+        margin-left: 7.5px;
+        margin-right: 7.5px;
```
Comment (stevenpetryk) [inline app/assets/stylesheets/footer/index.scss]: @maxcell could you just round these down to 7? Half-pixels are scurry

File: app/assets/stylesheets/footer/index.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      text-align: center;
+    }
+
+    .links {
+      float: none;
+      margin: 0 auto;
+      width: 50%;
+    }
+  }
+}
```
Comment (stevenpetryk) [inline app/assets/stylesheets/footer/index.scss]: This is all great!

File: app/assets/stylesheets/footer/index.scss (hunk 1 above)
Comment (maxcell) [inline app/assets/stylesheets/footer/index.scss]: :+1: Taken care of!

**Comment by maxcell** [comment]
It shouldn't conflict but do you want me to rebase this so it doesn't cry with your latest PR merge?

**Comment by stevenpetryk** [comment]
I'll check it out and make it jive with the new section.

**Comment by maxcell** [comment]
This is on Safari on Full Window. Any thoughts?
image

**Comment by maxcell** [comment]
Not quite yet, we need to fix this at 425px on Chrome.
image

And then in Safari basically throughout kek.
screen shot 2017-02-21 at 5 21 57 pm

We can leave this for now and just work out the content in a bit anyway. I can write up more issues if you'd like?

**Comment by stevenpetryk** [comment]
Sure thing. Safari display bugs should be handled last, mostly because browser-specific hacks will weigh down development if they're done while things are still really volatile.

**Comment by maxcell** [comment]
Okie dokie! I will make sure to attribute labels to Safari specific bugs. I will also be sure to attribute things back to this PR so we maintain their occurrence and hopefully their resolution in the future.

Beyond all of this, 🍏?

**Pull request merged**